project-chip/connectedhomeip
# [Telink]: Use separate OpenThread configs for apps

#### Problem
Need to use separate OpenThread configs for apps

#### Change overview
Used separate OpenThread configs for Telink apps

#### Testing
Tested manually with chip-tool.
Steps:

- Run: $ chip-tool pairing ble-thread <...>
- Wait till success
- Run: $ chip-tool accesscontrol write acl <...>
- Wait till success
- Run: $ chip-tool binding write binding <...>
- Wait till success
- Press manually switch button
- Check if end device perform command
